locust/runners.py

```diff
@@ -30,8 +30,17 @@
 from .dispatch import UsersDispatcher
 from .exception import RPCError, RPCReceiveError, RPCSendError
 from .log import get_logs, greenlet_exception_logger
-from .rpc import Message, rpc
-from .stats import RequestStats, StatsError, setup_distributed_stats_event_listeners
+from .rpc import (
+    Message,
+    rpc,
```

Comment:
Did you accidentally add a "magic" trailing comma here, or is your ruff not tuned to the same line length as the project? :)